### ISSUE TITLE
[New Resource] NIC Teaming (NetLBFO)

### ISSUE THREAD

**Issue by tysonjhayes** [PowerShell/xNetworking#80]
Add [Nic Teaming (NetLBFO)](https://technet.microsoft.com/en-us/library/jj130849(v=wps.630).aspx) to the list of resources.

**Comment by rchaganti** [comment]
Here it is: https://github.com/rchaganti/DSCResources/tree/master/cWindowsOS/DSCResources/cNetworkTeam

I just need to update it as per contribution guidelines here and add tests! :D

**Comment by rchaganti** [comment]
When I copy this new resource to my fork, what should I name it? Leave it as cNetworkTeam or name it MSFT_xNetworkTeam?

**Comment by tysonjhayes** [comment]
If the intention is to submit a PR to us then it should be renamed MSFT_xNetworkTeam. Though you bring up a good question that I've proposed elsewhere that we should drop the x/c nomenclature. So it would just be MSFT_NetworkTeam instead.  What are your thoughts?

**Comment by rchaganti** [comment]
Yes, I will submit a PR here. Given an option, I will remove x or c from it completely.

**Comment by PlagueHO** [comment]
The x/c isn't really that useful TBH - I thought x was for experimental and c was for community - but it seems like useless information to me. But to remove it now from all resources would mean all configs using them would need to be updated - which might not be ideal.

All my resources use c, but as I'm transferring them across to MSFT I'm changing them to x.

As long as the naming is consistent I'm happy :)

**Comment by rchaganti** [comment]
I didn't mean that we change existing resources. We can just drop the x/c prefix for all new resources.

**Comment by PlagueHO** [comment]
I just noticed that there has been some discussion already around dropping the x/c nomenclature in this issue:
https://github.com/PowerShell/DscResources/issues/27

It looks like the policy was to drop the x/c off the resources once ALL 'c' resources are integrated and all 'x' resources are adhering to the best practices policy.

I'm not certain of what the final plan is currently though as this discussion was during June last year.

I'd suggest we stick with the 'x' for now and when a final instruction is issued we could submit a PR to change all resources. This actually shouldn't be difficult as I've changed some of my resources from 'c' to 'x' without too much difficulty.

@tysonjhayes ?
